decimal.go

```diff
@@ -126,7 +126,7 @@ func New(value int64, exp int32) Decimal {
 			}
 		}
 		// when exp > 7, it would be greater than maxInt
```

Comment:
```suggestion
		// when exp > 6, it would be greater than maxInt
```

Reply:
it would be either `> 7` or `>= 6`

Reply:
@gyson you mean either `>= 7` or `> 6`?

Reply:
yep. @AlexandrosKyriakakis was right at first.